fedimintd/src/fedimintd.rs

```diff
@@ -32,9 +33,11 @@ use crate::attach_default_module_init_params;
 /// Time we will wait before forcefully shutting down tasks
 const SHUTDOWN_TIMEOUT: Duration = Duration::from_secs(10);
 
+/// Default meta value to use during config generation
```

Comment:
It's the name of a environent variable that allows passing extra metadata to pass to dkg.